client/my-sites/woocommerce/landing-page.tsx

```diff
@@ -190,13 +189,13 @@ const LandingPage: React.FunctionComponent< Props > = ( { siteId } ) => {
 
 	return (
 		<div className="landing-page">
-			<FixedNavigationHeader navigationItems={ navigationItems } ref={ headerRef }>
+			<NavigationHeader navigationItems={ [] } title="WooCommerce" ref={ headerRef }>
```

Comment:
```suggestion
			<NavigationHeader navigationItems={ [] } title={ __( "WooCommerce" ) } ref={ headerRef }>
```

Not sure if this changes anything for anyone but probably a good idea either way :smile:

Reply:
You're right, we never know what could happen 😉